manual/manual/cmds/unified-options.etex

```diff
@@ -169,6 +169,10 @@ chapter~\ref{c:intf-c}.
 Never use the "strip" command on executables produced by "ocamlc -custom",
 this would remove the bytecode part of the executable.
 \end{unix}
+\begin{unix}
+Security warning: never set the "setuid" or "setgid" bits on executables
```

Comment:
The double quotes around "setuid" causes  this word to be typeset in \tt font.  This is probably not what you want.  Consider ``setuid'' instead.  Likewise for setgid.